enterprise-suite/tests/e2e/cypress/support/navigation.ts

```diff
@@ -18,6 +18,10 @@ export class Navigation {
         cy.get(`.monitor-list .monitor-name[title="${monId}"]`, {timeout: 12000}).click();
     }
 
+    static clickMonitorByName(monId: string) {
+        cy.get(`.monitor-list .monitor-name[monitorname="${monId}"]`, {timeout: 12000}).click();
```

Comment:
I thought we said we'd use ```id``` ?

The name acts as a unique identifier (and the parameter in this method is monitor id)